src/test/java/andreeaG/tests/tema8/pageTests/BaseTest.java

```diff
@@ -0,0 +1,39 @@
+package andreeaG.tests.tema8.pageTests;
+
+import andreeaG.driver.BrowserManager;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BaseTest {
+
+    protected ChromeDriver driver;
+
+    @BeforeMethod(groups = "desktop")
+    public void desktopTest() {
+        driver= BrowserManager.createChromeDriverWithOptions();
+    }
+
+    @BeforeMethod(groups = "mobile")
```

Comment:
Use onlyForGroups instead of groups attribute. You are having one scenario for both groups in your tests and it's opening two instances of browsers when running tests in the actual configuration.